HISTORY.md

```diff
@@ -200,7 +200,7 @@ high number of columns or a high number of categorical values when using one hot
 
 ### Issues closed
 
-* `Boolean`, `Datetime` and `LabelEncoding` transformers fail with 2D `ndarray` - Issue [#160](https://github.com/sdv-dev/RDT/issues/160) by @pvk-developer
+* `Boolean`, `Datetime` and `LabelEncodingTransformer` transformers fail with 2D `ndarray` - Issue [#160](https://github.com/sdv-dev/RDT/issues/160) by @pvk-developer
```

Comment:
Should be `LabelEncoder` now right?

Reply:
I don't think so, since it's in the `HISTORY.md` file, so the naming should be as was at the time, right?